client/my-sites/migrate/section-migrate.scss

```diff
@@ -1,3 +1,17 @@
+$masterbar-height: 46px;
+
+@include breakpoint( '>660px' ) {
```

Comment:
I think what you really want to do here is to do vertical centering only if the layout _height_ is large enough. But this condition applies to the layout _width_. The width and height are not related to each other -- only loosely correlated.

Using `@media ( max-height )` or `@media ( min-height )` seems like a better option.

Reply:
I was relying on the loose correlation, but that would probably be better.